prebuilt/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../Makefile.versions.include
+include packages
+
```

Comment:
It might be less code and better readability if you defined a variable like `PREBUILT_OUTPUT_DIR` instead of using of `$(abspath ../output/prebuilt)`